Eastern-Research-Group/mywaterway
# Feature/160 monitoring bubble size clustering

## Related Issues:
* https://jira.epa.gov/browse/HMW-160

## Main Changes:
* Updated the monitoring location layer, such that the graphics are sized based on the percentile of the total measurements.
* Added a temporary switch for toggling the point clustering on/off. 

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/monitoring
2. Verify the monitoring station graphics are sized differently based on the percentile rank of measurement count
3. Verify the `TEMP Monitoring Clustering` switch is visible above the search bar
4. Toggle the `TEMP Monitoring Clustering` switch on
5. Verify the monitoring stations are now clustered
6. Toggle the `TEMP Monitoring Clustering` switch off
7. Verify the monitoring stations are no longer being clustered
8. Click on the `Drinking Water` tab
9. Verify the `TEMP Monitoring Clustering` switch is no longer visible

## TODO:
- [x] PR #601 needs to be merged in before this one can be merged.
- [x] PR #603 needs to be merged in before this one can be merged.
- [ ] Determine whether or not EPA would like to use point clustering or sizing points based on percentile rank and remove the `TEMP Monitoring Clustering` switch.
